build.sh

```diff
@@ -15,6 +15,22 @@ unset CALLBACK
 
 # Run build process based on configuration files
 
+# load nvm
+. "$NVM_DIR/nvm.sh"
```

Comment:
I don't think this is necessary now that we are using bash with the `login` option